src/pages/resources/testnet/upgrades/v35.mdx

```diff
@@ -53,16 +55,16 @@ cp -r ~/.axelar_testnet/.core/data ~/.axelar-lisbon-3-upgrade-0.34/.core/data
   excluding them from any data backups.
 </Callout>
 
-4. Restart your `axelard` node with the new `v0.35.3` build.
-5. If you're a validator also restart `vald` with `v0.35.3` and `tofnd` with `v0.10.1`.
+4. Restart your `axelard` node with the new `v0.35.5` build.
+5. If you're a validator also restart `vald` with `v0.35.5` and `tofnd` with `v0.10.1`.
```

Comment:
same, don't switch here. We need to update the resources page that states the latest version. See [here](https://docs.axelar.dev/resources/contract-addresses/testnet)